rollout/experiment.go

```diff
@@ -64,9 +64,7 @@ func GetExperimentFromTemplate(r *v1alpha1.Rollout, stableRS, newRS *appsv1.Repl
 			Name:     templateStep.Name,
 			Replicas: templateStep.Replicas,
 		}
-		if templateStep.Weight != nil {
```

Comment:
Feels like we can use weight: 0 to keep creating the service, if automated creation is rolled back